src/utils/constants/links.ts

```diff
@@ -8,14 +8,16 @@ const URL_PARAMETERS = Object.freeze({
     WRAPPED: 'vaultWrapped'
   },
   TRANSACTION_HASH: 'transactionHash',
-  STRATEGY_TYPE: 'strategyType'
+  STRATEGY: {
+    TYPE: 'strategyType'
+  }
```

Comment:
What is the purpose of putting this into object when there's only single property?

Reply:
Because the objective is having the key as the page and the paths as values and keep it consistent across the object as in the vault.

Reply:
Ah, in that case we should also change `TRANSACTION_HASH` for Transaction page to keep this consistent.

Reply:
Correct, its an old key so I will change. Good catch